ivr/src/main/resources/webapp/js/controllers.js

```diff
@@ -153,6 +153,19 @@
             }
         }
 
+        $scope.setNewDefaultConfig = function() {
+            var i;
+            for (i = 0 ; i < $scope.configs.configList.length ; i = i + 1) {
+                if ($scope.configs.configList[i].name === $scope.configs.defaultConfig) {
+                    return;
+                }
+            }
+            if ($scope.configs.configList.length > 0) {
+                $scope.configs.defaultConfig = $scope.configs.configList[0].name;
+                $scope.defaultConfigIndex = 0;
```

Comment:
This variable is unnecessary, you are assign only value to it but nothing is done with this value